crates/swc/src/config/mod.rs

```diff
@@ -1021,6 +1028,9 @@ pub struct JscConfig {
 
     #[serde(default)]
     pub lints: LintConfig,
+
+    #[serde(default)]
+    pub preserve_dropped_comments: bool,
```

Comment:
is it necessary? 
I suppose swc should always preserve comments.
And It's ok to remove Comments at the minify phase.

Reply:
It's because this patch tries to preserve comments on dropped nodes.

Btw, I think `preserve_all_comments` is better name. (Even if it does not preserve all comments)

Reply:
@magic-akari I added this because it was requested here: https://github.com/swc-project/swc/issues/2964#issuecomment-996189885

@kdy1 : agreed, see https://github.com/swc-project/swc/pull/3815/commits/3ce4af74d690bfad24dc40f33526ee5b1b2d8fa5